test/aliases.js

```diff
@@ -0,0 +1,59 @@
+import { join } from 'path'
+import { version } from 'process'
+
+import test from 'ava'
+
+import { resolveVersionRangeAlias } from '../src/aliases.js'
+
+const resolveInFolder = (versionRange, folder) =>
+  resolveVersionRangeAlias(versionRange, {
+    cwd: join(__dirname, 'fixtures', folder),
```

Comment:
```suggestion
    cwd: `${__dirname}/fixtures/${folder}`,
```

Should work because `fs.*` core methods (which `find-up` relies on) work with forward slashes even on Windows.